server/src/app.js

```diff
@@ -9,8 +9,8 @@ var indexRouter = require('./routes/index');
 var app = express();
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'pug');
+// app.set('views', path.join(__dirname, 'views'));
```

Comment:
Please remove comments.

Reply:
Yes, view was needed by @nitishmittal1990 for some testing with babel.